File: easy/Swap-Case/Swap-Case.java
import java.io.*;
public class Main {
    public static void main (String[] args) throws IOException {
        File file = new File(args[0]);
        BufferedReader buffer = new BufferedReader(new FileReader(file));
        String line;
        while ((line = buffer.readLine()) != null) {
            line = line.trim();
            // Process line of input Here
            ShowInvertedSentence(line);
        }
    }
    
     public static void ShowInvertedSentence(String Rawline){
            char[] line = Rawline.toCharArray();
            String result = "";
            for(int i=0; i<line.length;i++) {
                result = result + InvertCase(String.valueOf(line[i]));
            }
            System.out.println(result);
        }
        
        public static String InvertCase(String ch){
            if (ch.matches("[a-z]")) return ch.toUpperCase();
            if (ch.matches("[A-Z]")) return ch.toLowerCase();
            return ch;
        }
    
}

File: easy/Swap-Case/Swap Case.cs
using System;
using System.IO;
using System.Text;
using System.Collections.Generic;
using System.Text.RegularExpressions;

class Program
{
    static void Main(string[] args)
    {
        using (StreamReader reader = File.OpenText(args[0]))
        while (!reader.EndOfStream)
        {
            string line = reader.ReadLine();
            if (null == line)
                continue;
            // do something with line
            ShowInvertedSentence(line);
        }
    }
    
    
    public static void ShowInvertedSentence(string Rawline){
            StringBuilder line = new StringBuilder(Rawline);
            for(int i=0; i<line.Length;i++) line[i] = InvertCase(line[i]);
            System.Console.WriteLine(line);
        }
        
        public static char InvertCase(char ch){
            char result = ch;
            if (Regex.IsMatch(ch.ToString(),"[a-z]")) result = Char.ToUpper(ch);
            if (Regex.IsMatch(ch.ToString(),"[A-Z]")) result = Char.ToLower(ch);
            return result;
        }
    
}
